CHANGELOG.md

```diff
@@ -1,6 +1,16 @@
 # Changelog
 
-## v0.1.1 (24 Nov 2022)
+## [Unreleased]
+
+### Added
```

Comment:
To me, the changelog is more for customers. Especially it will also be displayed in the marketplace and the extension page in VS Code.
Reading the "Keep a Changelog", it says
- `Added` for new features.
- `Changed` for changes in existing functionality.

So my feeling is that this change doesn't need to be documented here

Reply:
Makes sense to me :)